realworld/README.md

```diff
@@ -0,0 +1,35 @@
+# <img src="logo.png" width="48"> RealWorld Example App
+
+> ### Dojo 2 codebase containing real world examples (CRUD, auth, advanced patterns, etc) that adheres to the [RealWorld](https://github.com/gothinkster/realworld) spec and API.
+
+### [Demo](https://dojo.github.io/examples/realworld/)&nbsp;&nbsp;&nbsp;&nbsp;
+
+
+This codebase was created to demonstrate an application built with Dojo 2 including CRUD operations, authentication, routing, pagination, and more.
+
+For more information on how to this works with other frontends/backends, head over to the [RealWorld](https://github.com/gothinkster/realworld) repo.
+
+
+# How it works
+
+Dojo 2 RealWorld using @dojo/widget-core, @dojo/routing and @dojo/stores.
+
+For more information on Dojo 2 visit [dojo.io](https://dojo.io)
+
+# Getting started
+
+You can view a live demo over at https://dojo.github.io/examples/realworld/
```

Comment:
You can view a live demo over at https://dojo.github.io/examples/realworld/ -> View the [Dojo 2 RealWorld live demo](https://dojo.github.io/examples/realworld/).